pages/index.vue

```diff
@@ -101,11 +101,9 @@
         </v-card>
       </v-col>
       <v-col cols="12" md="4">
-        <ul v-if="loaded" style="list-style-type:none">
-          <li>
-            <h3 class="h2 mt-2 mb-2">{{ $t('home.news.title') }}</h3>
-          </li>
-          <li v-for="(newsItem, i) in recentNewsItems" :key="i">
+        <h3 class="h2 mt-2 mb-2">{{ $t('home.news.title') }}</h3>
+        <span v-if="loaded">
+          <span v-for="(newsItem, i) in recentNewsItems" :key="i">
```

Comment:
Think we can use `<v-list-item>`? Like in two line item of the Codepen example in https://vuetifyjs.com/en/components/lists/#usage

Reply:
Using <v-list-item> indents the items in the section and it looks a bit awkward, unless maybe there is a way to remove the indentation
<img width="207" alt="Capture" src="https://user-images.githubusercontent.com/83600460/122117465-e0abc580-cdf4-11eb-82ff-48ffe9f9bcbd.PNG">

Reply:
A bit unsure with this edit, since using the list tags indents the items and makes the items misalign with the headers:
<img width="207" alt="Capture" src="https://user-images.githubusercontent.com/83600460/122235392-a7726480-ce8b-11eb-9e06-949712a131df.PNG">